src/index.css

```diff
@@ -0,0 +1,74 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  
+  font-family: sans-serif;
+  margin: 3rem;
+  background-color: #dfdfdf;
+}
+
+h1, h2 {
+  color: #333333;
+}
+
+.btn {
```

Comment:
I prefer using long/full words for class names, i.e. `button` not `btn`.
Optional